tests/Feature/ResetPasswordTest.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Tests\Feature;
+
+use App\Models\ResetPassword;
+use App\Models\User;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Hash;
+use Illuminate\Support\Str;
+use Tests\TestCase;
+
+class ResetPasswordTest extends TestCase
+{
+    use RefreshDatabase;
+    /** @test */
+    public function test_resetPassword_method()
+    {
+        $this->artisan('passport:install');
+        $pass = 'user1';
+        $password_hash = Hash::make($pass);
+        $user = User::factory()->create([
+            'email' => 'user1@gmail.com',
+            'password' => $password_hash
+        ]);
+        $response = $this->json('POST', 'api/reset_password',[
+            'email' => $user->email
+        ])->assertStatus(200);
+        $response->assertJsonStructure(['result', 'message']);
```

Comment:
add Mail::assertSent(ResetPassword::class); after request, to check that mail has been sent